### ISSUE TITLE
1.8.7 support?

### ISSUE THREAD

**Issue by a-warner** [heroku/legacy-cli#1100]
Does the heroku client still support ruby 1.8.7? It looks like [this commit](https://github.com/heroku/heroku/commit/f4e1e2123de2758666a3dcdec7f586d97c18efd6) merged a file which uses 1.9+ hash syntax, which means the client just blows up on every command now..not great.

**Comment by dickeyxxx** [comment]
@a-warner just want to warn you that I am probably going to land #1479 tomorrow. So long as you are on the toobelt it should be using 1.9 though. Hopefully it works without issues, but let me know if you see any.

**Comment by a-warner** [comment]
@dickeyxxx thanks for letting me know! I'm on the toolbelt these days so I think I'm ok (but the toolbelt still says ruby/1.8.7?)

**Comment by dickeyxxx** [comment]
uh oh, you sure you're on the toolbelt? what does `heroku version` print?

**Comment by a-warner** [comment]
```ruby
$ heroku --version
heroku-toolbelt/3.30.3 (i686-darwin13.1.0) ruby/1.8.7
```

**Comment by dickeyxxx** [comment]
definitely want to make sure we can fix this before I push anything. (There are a couple security CVEs out for one of the gems, that's why I need to update this). Do you know if you installed the toolbelt with the standalone or the homebrew install? If so, you should be able to install the pkg version from here and be on 1.9 https://toolbelt.heroku.com/osx

**Comment by a-warner** [comment]
I've got the homebrew version, is that updated yet?

**Comment by dickeyxxx** [comment]
ah so homebrew uses whatever ruby is on the system path, you must be on an older version of OSX that shipped with 1.8.7. If you run `brew install ruby` or use rbenv or rvm you should be able to run a newer version.

Would you mind trying `brew install ruby` to see if that puts you on a newer version?

**Comment by dickeyxxx** [comment]
just so this isn't a totally useless exercise for you, you should get a decent speedup :)
